src/System.Security.Cryptography.OpenSsl/src/System.Security.Cryptography.OpenSsl.csproj

```diff
@@ -13,17 +13,13 @@
     <CLSCompliant>false</CLSCompliant>
     <PackageTargetFramework>netstandard1.4</PackageTargetFramework>
     <NuGetTargetMoniker>.NETStandard,Version=v1.4</NuGetTargetMoniker>
-    <!-- Temporarily remove the RID from this package until we have a  
-         cross platform API to use RSA https://github.com/dotnet/corefx/issues/2953 -->  
-    <PackageTargetRuntime Condition="'$(TargetGroup)' == ''">unix</PackageTargetRuntime>  
+    <PackageTargetRuntime Condition="'$(TargetsUnix)' == 'true'">unix</PackageTargetRuntime>
+    <PackageTargetRuntime Condition="'$(TargetsWindows)' == 'true'">win</PackageTargetRuntime>
```

Comment:
I'm avoiding the use default properties for now.  It created confusion because folks were using that and overriding it.  This is more WYSIWYG.  I was talking with @weshaggard about moving more of this boilerplate to the dir.props that's translating OSGroup/TargetGroup to various settings.